client/components/faq/index.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Internal dependencies
+ */
+
+import FAQ from './faq';
+import FAQItem from './faq-item';
+
+export default { FAQ, FAQItem };
```

Comment:
Perhaps, not sure :) We don't have a lot of cases where it is imperative to import the two files. I wasn't saying we should change it, just that it'd be a first case.